EasyInventory/src/main/java/com/easyinventory/enterprise/controller/EasyInventoryController.java

```diff
@@ -18,45 +18,45 @@
 public class EasyInventoryController {
 
     @Autowired
-    EasyInventoryService easyInventoryService;
+    private EasyInventoryService easyInventoryService;
     //Endpoints for the categories
 
     //Retrieves all the categories from the database
-    @RequestMapping(value="/category/all", method = RequestMethod.GET)
+    @GetMapping("/category/all")
```

Comment:
Instead of that long-winded @RequestMapping, SpringBoot has built in annotations for all types of requests. For this example, @GetMapping has the same functionality with less typing than your current implementation.

Reply:
Good idea.  Less typing == less chance for error.